.gitignore

```diff
@@ -2,6 +2,8 @@
 .coverage
 .eggs/
 .env
+.vscode/
+.python-version
```

Comment:
Please back these changes out. Editor-specific or personal ignores are best put into a `~/.gitignores` file which can be configured with 

```
git config --global core.excludesFile $HOME/.gitignores
```

Reply:
I will. Thank you for the suggestion.